src/Attendance/AttendanceOtherForm/AttendanceOtherForm.styles.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components'
+
+// eslint-disable-next-line import/prefer-default-export
+export const ConditionalRadio = styled.div`
+  margin-bottom: 15px;
+  padding-left: 33px;
+  border-left: 4px solid #b1b4b6;
```

Comment:
Couldn't find a mapping for this colour in react govuk so grabbed it from the condition radio style